### PR TITLE
Nellie edits

### PR DESCRIPTION
**Pull request by marklhc**
Need review before propagating the change

### CODE REVIEW

File: members.Rmd [hunk 1]
```diff
@@ -48,64 +53,60 @@ I am interested in all kinds of statistical methods, but currently my research a
...
 
 - Email: ozcan@usc.edu
 - Personal website: https://www.linkedin.com/in/meltemozcan/ 
+- Github: https://github.com/meltemozcan
 
 ***
 
 ## Undergraduate Students
```
Comment (meltemozcan) [inline members.Rmd:98]: @Nellie021231 You should add a blurb for yourself here too!

File: members.Rmd [hunk 2]
```diff
@@ -48,64 +53,60 @@ I am interested in all kinds of statistical methods, but currently my research a
...
 
-### Hailin Yue (Haley)
-
-I am a fourth-year undergraduate student majoring in economics, with a minor in psychology and data analytics. I am interested in applying quantitative reasoning to study mental health and to optimize business decision I hope to pursue a graduate degree after graduation, but am not sure on what subject yet. Outside of academia, I enjoy reading, painting, listening to music, and watching TV dramas.
-
-- Email: hailinyu@usc.edu
+### Yixiao
```
Comment (meltemozcan) [inline members.Rmd:99]: @Nellie021231 let's add last names for Yixiao and Kazia

**Comment by meltemozcan** [comment]
Great job with all the updates @Nellie021231! Thanks for your hard work. I have reviewed the changes and all looks good with the PR itself but let's check in with @marklhc and @winniewytse about the deployment check related issues before merging. 

One question for @marklhc, should the PR be nellie_edits -> master or nellie_edits -> devel? devel is 5 commits ahead of master. 

As a side note for @Nellie021231, I created a branch off of nellie_edits to automate the extraction of image paths for the slideshow from the images folder as previously discussed with Mark. Once we determine what's happening with deployment we can merge my branch into yours and then merge into master (or devel, depending on Mark's reply to the above note).

File: members.Rmd [hunk 3]
```diff
@@ -51,61 +56,57 @@ I am interested in all kinds of statistical methods, but currently my research a
...
+I am a fourth-year Ph.D. student in Quantitative Psychology. I am co-advised by Dr. Hok Chio(Mark) Lai and Dr. Rand Wilcox. My current research interests are measurement bias in surveys/questionnaires and robust statistical models, such as hierarchical linear model(linear mixed effects model), latent variable model (structural equation modeling), etc. I hope to use psychometric methods and statistical models to explore research questions related to fair and accurate psychological measurement. My main project focuses on quantifying the practical impact of item bias(noninvariance) on total scale scores. My secondary project evaluates the impact of two-level heteroscedasticity on small sample corrections for fixed-effects inferences. In addition, I am involved in projects on bootstrapping for multilevel models, multilevel regression and poststratification on subnational predictions with language data.
 
 - [CV](static/cv/YichiZhang_CV_20220206.pdf)
 - Email: yzhang97@usc.edu
+- Personal website: https://yichiyczhang.rbind.io/about.html
```
Comment (YichiZhang2024) [inline members.Rmd:73]: Should we use consistent formatting? So [Personal website](https://yichiyczhang.rbind.io/about.html)?

File: publications.Rmd [hunk 4]
```diff
@@ -48,6 +51,22 @@ for (yeari in as.character(2022:2019)) {
...
+
+Zhang, G., & Lai, M. H. C. (2022, July 11-15). *A systematic review of measurement invariance research of the CES-D across gender: calculation and report of effect size* [Oral presentation]. Annual Meeting of the Psychometric Society (IMPS), Bologna, Italy. https://www.psychometricsociety.org/sites/main/files/file-attachments/imps2022-version-7-abstract.pdf
+
+Tse, W. W., & Lai, M. H. C. (2022, July 11-15). *Incorporating information from historical data for power analysis: A hybrid classical-Bayesian approach for multilevel studies *[Oral presentation]. Annual Meeting of the Psychometric Society (IMPS), Bologna, Italy. https://www.psychometricsociety.org/sites/main/files/file-attachments/imps2022-version-7-abstract.pdf
+
+Zhang, Y., & Lai, M. H. C. (2022, July 11-15). *Bayesian region of measurement equivalence (ROME) approach with alignment *[Oral presentation]. Annual Meeting of the Psychometric Society (IMPS), Bologna, Italy. https://www.psychometricsociety.org/sites/main/files/file-attachments/imps2022-version-7-abstract.pdf
+
```
Comment (YichiZhang2024) [inline publications.Rmd:63]: I realize I forgot to add this conference presentation: 
Zhang, Y., & Lai, M. H. C. (2022, July 11-15). *Evaluating standard error estimators and resampling method for multilevel models on small samples with heteroscedasticity and unbalanced cluster sizes *[Poster presentation]. Annual Meeting of the Psychometric Society (IMPS), Bologna, Italy.

**Review by YichiZhang2024 — APPROVED** [review]

**Comment by marklhc** [comment]
I resolved the merge conflicts. @meltemozcan should we wait for #14 to finish first before merging this?

**Comment by meltemozcan** [comment]
> I resolved the merge conflicts. @meltemozcan should we wait for #14 to finish first before merging this?

@marklhc Thank you Mark! I think we can complete the merge in #14 (nellie edits from mel_nellie_edits) first once the code has been reviewed. It's a few lines of code so I hope it wouldn't take much time. I can send a review request to you for that pull request if you would like. Once we complete #14 we can merge this too. Please let me know if you'd like me to go ahead with either of the merges.

**Pull request merged**